packages/host/app/router.ts

```diff
@@ -20,5 +20,7 @@ Router.map(function () {
     this.route('card', { path });
     this.route('code', { path: `${path}/code` });
   }
-  this.route('card', { path: '/*path' });
+  this.route('card', { path: '/*path' }, function () {
+    this.route('operator-mode');
```

Comment:
I think we got rid of this route. Must be left over from a previous branch

Reply:
Yeah the changes are in main. The branch needs a rebase